src/System.Private.CoreLib/src/System/StringComparison.cs

```diff
@@ -16,12 +16,13 @@
 
 namespace System
 {
+    [Serializable]
     public enum StringComparison
     {
         CurrentCulture = 0,
         CurrentCultureIgnoreCase = 1,
-        //InvariantCulture = 2,
-        //InvariantCultureIgnoreCase = 3,
+        InvariantCulture = 2,
+        InvariantCultureIgnoreCase = 3,
```

Comment:
Do these changes need to be paired with work in our string comparison algorithms?